src/cdk/a11y/_a11y.scss

```diff
@@ -28,4 +28,10 @@
   @media (-ms-high-contrast: $target) {
```

Comment:
can we detect this in JS and apply the appropriate class instead so that we don't need to duplicate all of the styles?

Reply:
We potentially don't need the `@media` query at all if the class works for IE11 and Edge, but I don't have a Windows machine available to confirm at the moment (our test workstation is busted). I'll chat with Kristiyan about it

Reply:
Sounds good. If what you have currently doesn't work we can probably add some extra logic to detect on IE/Edge which would likely wind up smaller than duplicating all the CSS

Reply:
I wasn't super concerned because the high-contrast styles tend to be pretty small and relatively infrequent, but agreed that it's better to avoid if possible.